src/main/java/edu/harvard/iq/dataverse/batch/util/LoggingUtil.java

```diff
@@ -49,6 +52,21 @@ public static void saveJsonLog(String jobJson, String logDir, String jobId) {
 		    logger.log(Level.SEVERE, "Error saving json report: " + e.getMessage());
 	    }
     }
+    
+    public static void saveDraftEditLog(String changes, String logDir, String identifier, String datasetId) {
```

Comment:
I think as we add more logs, we should centralize this logic to be usable for all future logging. Having a "saveLog" that takes a relative directory name and a string to log would be more reusable. The code for creating the log name can be held in the bean that needs this.

This is somewhat self-serving as I'm going to write my own log that does almost all the same things.